UI/MVUX/src/MVUX/Presentation/FeedViewSample/FeedViewPage.xaml

```diff
@@ -0,0 +1,119 @@
+﻿<Page x:Class="MVUX.Presentation.FeedViewSample.FeedViewPage"
```

Comment:
For this sample, I think that we should put the 3 templates in page static resources and re-use them on all `FeedView`. It would reduce the amount of XAML and better explain that this is only to display the different states.